Dockerfile

```diff
@@ -0,0 +1,10 @@
+FROM golang:alpine as reboot-api-build
```

Comment:
Please add version linkage.

Reply:
Actually, consider modeling things based on https://github.com/m-lab/pusher/blob/master/Dockerfile

Reply:
Done. It took me a bit to figure out why the executable built without `CGO_ENABLED=0` wasn't running on the alpine container. I considered using golang:1.12-alpine for building instead of disabling CGO. Any explicit preference for using `golang:1.12` vs `golang:1.12-alpine` ?